projects/valtimo/process-management/src/lib/process-management-builder/process-management-builder.component.html

```diff
@@ -27,6 +27,7 @@
         <button
           class="btn btn-success btn-space"
           (click)="deploy()"
+          [hidden]="isReadOnlyProcess$ | async"
```

Comment:
I see in multiple places you used `[hidden]` instead of `*ngIf`. Please switch to `*ngIf`, it offers performance benefits: https://stackoverflow.com/questions/43034758/what-is-the-difference-between-ngif-and-hidden

Reply:
did not know that ... I will make the changes